src/pkg/fault/fault.go

```diff
@@ -67,24 +88,16 @@ func (e *Bus) Failure() error {
 // recoverable points of processing.  This is often during
 // iteration where a single failure (ex: retrieving an item),
 // doesn't require the entire process to end.
+// If the bus is a local instance, this only returns the
+// local recovered errors, and will not return parent data.
 func (e *Bus) Recovered() []error {
 	return slices.Clone(e.recoverable)
 }
 
-// Skipped returns the slice of items that were permanently
-// skipped during processing.
-func (e *Bus) Skipped() []Skipped {
```

Comment:
Can we please keep all functions for a struct in the same file? It makes it easer to determine what the full API of the struct is. The `Skipped()` function is still related to `Bus`, it just deals with a different aspect of it. There's already quite a few files in the codebase and splitting everything into the smallest possible unit per file just adds overhead

Reply:
Worth a design discussion sometime.  I, personally, find that splitting packages into small files representing semantic chunks is smaller cognitive load (I assume that's what you mean by 'overhead') than maintaining fewer, larger files.

For now, I'm happy to move these methods back.